accessibility/accessibility-checklist.md

```diff
@@ -39,6 +39,30 @@ Resources:
 * [Tool: HTML5 accessibility feature support for sections](http://www.html5accessibility.com/#sections)
 
 
+### Provide a Skip Navigation option
```

Comment:
```suggestion
### Provide skip links to bypass repetitive content
```

Suggesting this instead of "skip navigation", because there are other things you might want to add a bypass to (e.g. very long, tiresome widgets with 100s of tab stops)